server/fleet/hosts.go

```diff
@@ -652,28 +652,68 @@ func (h *Host) FleetPlatform() string {
 	return PlatformFromHost(h.Platform)
 }
 
-// HostLinuxOSs are the possible linux values for Host.Platform.
-var HostLinuxOSs = []string{
-	"linux", "ubuntu", "debian", "rhel", "centos", "sles", "kali", "gentoo", "amzn", "pop", "arch", "linuxmint", "void", "nixos", "endeavouros", "manjaro", "opensuse-leap", "opensuse-tumbleweed",
+// LinuxOS All Linux OS grouped by what they are based on ("rpm", "debian", etc)
+var LinuxOS = map[string][]string{
+	"rpm": {
+		"centos",
+		"rhel",
+		"amzn",
+		"opensuse-leap",
+		"opensuse-tumbleweed",
+		"sles",
```

Comment:
Is it possible to have `deb_packages` on these systems? (Same with the debian list below, could those hosts have `rpm_packages`?)